packs/loaders/productListiningPage.ts

```diff
@@ -79,12 +74,43 @@ const loaders = async (
   );
 };
 
+const getSearchPageParams = (url: URL, extraParams: Props): PLPPageParams => {
+  const {
+    nome,
+    id,
+    idColecaoProdutos,
+    filtrosDinamicos,
+    somenteCronometrosAtivos,
+  } = extraParams;
+
+  return {
+    productApiProps: {
+      nome: url.searchParams.get("termo") ?? nome ?? "",
+      id: !id?.length ? undefined : id,
```

Comment:
prefira tratar as coisas pelo caso positivo, fica mais facil de entender em logicas complexas
```suggestion
      id: id?.length ? id : undefined,
```